### ISSUE TITLE
Typescript issue (solves deleting internal index.d.ts)

### ISSUE THREAD

**Issue by MichelDiz** [oblador/react-native-collapsible#174]
This thing It's a bit annoying, how to standardize/solves?

```typescript
[ts]
Type '{ activeSection: boolean; sections: { title: string; content: string; }[]; renderHeader: (section...' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<Accordion> & Readonly<{ children?: ReactNode; }> &...'.
  Type '{ activeSection: boolean; sections: { title: string; content: string; }[]; renderHeader: (section...' is not assignable to type 'Readonly<AccordionProps>'.
    Property 'renderSectionTitle' is missing in type '{ activeSection: boolean; sections: { title: string; content: string; }[]; renderHeader: (section...'.
(JSX attribute) duration: number
```

![captura de tela 2018-04-22 as 19 49 26](https://user-images.githubusercontent.com/6570955/39100853-f502394c-4666-11e8-8e3e-99e430f21a62.png)

PS. 	// tslint:disable was just a try.

**Comment by iRoachie** [comment]
Sorry about that, just made a fix in https://github.com/oblador/react-native-collapsible/commit/cb934c53495bb5eec215bd7e8311ed6b6f02d8b7. Will post version in couple hours.

**Comment by iRoachie** [comment]
Released in [`0.11.1`](https://github.com/oblador/react-native-collapsible/releases/tag/v0.11.1)
